bambi/tests/test_model.py

```diff
@@ -380,6 +382,17 @@ def test_cell_means_with_random_intercepts(crossed_data):
     priors1 = {x.name:x.prior.args['sd'].args for x in model1.terms.values() if x.random}
     assert set(priors0) == set(priors1)
 
+    # test summary
+    fitted.summary()
+    fitted.summary(exclude_ranefs=False)
```

Comment:
We should probably create a new `test_results` module, and add a fixture with fake, deterministically generated `Model` and PyMC3 `MultiTrace` inputs, so that we can test all of these methods in a modular way. We definitely want more assertions here.